src/containers/ConversionForm/index.tsx

```diff
@@ -144,6 +145,16 @@ const ConversionForm = ({
         }
     };
 
+    const renderUploadFile = () => {
+        if (fileToConvert) {
+            return (
+                <span className={styles.renderedFileName}>
+                    {fileToConvert.name}
+                </span>
+            );
+        }
+    };
+
```

Comment:
Should this have a return type if `fileToConvert` is null/undefined?

Reply:
Changed to always return the span and use a ternary to grab the filename if there is a `fileToConvert`